docs/cli.rst

```diff
@@ -1220,10 +1224,17 @@ facilitates rapid iterations, inspections of any pipeline as well as debugging.
 
     $ nextflow run main.nf -with-report
 
-- Invoke the nextflow pipeline execution with a custom queue size. By default, the value of **queue-size** is the same as the number of available CPUs. ::
+- Execute a pipeline with a custom queue size. By default, the default queue size is the number of available CPUs. ::
 
     $ nextflow run nextflow-io/hello -qs 4
 
+- Execute a pipeline locally with more concurrent tasks than the number of available CPUs. ::
+
+    $ nextflow run main.nf -executor.cpus 100 -ps 100 -qs 100
```

Comment:
This can be very inefficient why CPU bound tasks. Why advise it?

Reply:
It's meant for small local tasks that use less than 1 CPU. It's the best way to handle this case until we support fractional CPUs.

Reply:
See also this Slack thread: https://nextflow.slack.com/archives/C02T98A23U7/p1656081406989489

Reply:
I have updated this PR and clarified when this is appropriate to use.